controller/src/action/action_repository.rs

```diff
@@ -33,23 +84,100 @@ impl ActionRepository {
             status
         )
         .fetch_one(self.pool.as_ref())
-        .await
-        .unwrap();
+        .await?;
+
+        for command in &commands {
```

Comment:
A command is an entity that can be linked to an action, and the creation of an action must enable an action to be created.
So in action_service it is logical to find the creation of an action with the subsequent creation of a command.
But not in a repository. You would need to have a `command_repository` called in the `action_service`.

Reply:
I feel like it would be a bad idea to create such a repository since commands and actions are too much linked. Also keep in mind that a command is just a one line thing so I don't want to add a whole dedicated service to it

Reply:
Command is still a model, so if I decide to delete a command or change something else, what do I do?

Reply:
Ok I'll add it